website/docs/docs/core/connect-data-platform/risingwave-setup.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "RisingWave setup"
+id: "risingwave-setup"
+description: "Read this guide to learn about how to set up RisingWave in dbt."
+meta:
+  maintained_by: RisingWave
+  pypi_package: 'dbt-risingwave'
+  authors: 'Dylan Chen'
+  github_repo: 'risingwavelabs/dbt-risingwave'
+  min_core_version: 'v1.6.1'
+  min_supported_version: ''
+  cloud_support: Not Supported
+  slack_channel_name: '#all-things-risingwave'
```

Comment:
This was actually for our dbt Slack (through definitely unclear in naming). To align with our other adapters,  
```suggestion
  slack_channel_name: 'N/A'
```